tools/mason/MasonBuild.chpl

```diff
@@ -114,9 +114,10 @@ proc compileSrc(lockFile: Toml, binLoc: string, show: bool,
   if isFile(pathToProj) {
     var command: string = 'chpl ' + pathToProj + moveTo + ' ' + ' '.join(compopts);
     if release then command += " --fast";
+    if sourceList.numElements > 0 then command += " --main-module " + project;
```

Comment:
Is there any downside to always throwing the flag `--main-module`?

Reply:
Assuming that the module name and source file name match, no downside except being unnecessary.

Reply:
I'm in favor of always throwing it to reduce additional code-complexity of this `if` statement, but it's trivial enough that I'll leave it up to you.

Reply:
I think it's preferable to not use --main-module when it's not needed from a users perspective.  It just makes the command unnecessarily more complicated.

The added complexity is less than one line of code, so I'd prefer to leave it.

Reply:
> but it's trivial enough that I'll leave it up to you.

Sounds good.